PWGHF/Tasks/taskD0.cxx

```diff
@@ -228,28 +228,30 @@ struct TaskD0MC {
         registry.fill(HIST("hCPARecBg"), candidate.cpa());
         registry.fill(HIST("hEtaRecBg"), candidate.eta());
       }
+      auto massD0 = InvMassD0(candidate);
+      auto massD0bar = InvMassD0bar(candidate);
+      auto candidatept = candidate.pt();
+      auto candidaterapidity = YD0(candidate);
```

Comment:
* Use a consistent naming pattern:
  * `massD0` follows `<object><attribute>` (to be used),
  * whereas `candidatept` follows `<attribute><object>` (to be avoided)
* Follow the naming conventions (camelCase).